src/redux/features/plantings/plantingsSelectors.ts

```diff
@@ -89,24 +91,22 @@ export const searchPlantingProgress = createSelector(
     (state: RootState, query: string, plantingCompleted?: boolean) => plantingCompleted,
   ],
   (plantingProgress, query, plantingCompleted) => {
-    return plantingProgress
-      ?.filter((planting) => planting.totalPlants > 0)
-      .reduce((acc, curr) => {
-        const { siteId, siteName, totalPlants, reported } = curr;
-        if (reported && reported.length > 0) {
-          reported?.forEach((progress) => {
-            const matchesQuery = !query || regexMatch(progress.subzoneName, query);
-            const matchesPlantingCompleted =
-              plantingCompleted === undefined || progress.plantingCompleted === plantingCompleted;
-            if (matchesQuery && matchesPlantingCompleted) {
-              acc.push({ siteId, siteName, totalPlants, ...progress });
-            }
-          });
-        } else if (!query && plantingCompleted === undefined) {
-          acc.push({ siteId, siteName, totalSeedlingsSent: totalPlants });
-        }
-        return acc;
-      }, [] as any[]);
+    return plantingProgress?.reduce((acc, curr) => {
```

Comment:
this was a one line change (removed line 93 in previous code).. code formatter made the rest of the changes